@css/main_style.css

```diff
@@ -303,11 +796,122 @@ div.main-body {
 }
 
 #section_2 {
-  background-image: url("../@images/section-2_background.jpeg");
   background-attachment: scroll;
   background-repeat: no-repeat;
   background-size: cover;
   height: 588px;
   padding-left: 40px;
   padding-right: 40px;
 }
+
+#about-myself-layout {
+  padding: 0 0;
+  gap: 24px;
+}
+
+#about-myself {
+  padding: 0 24px 114px 24px;
+  align-items: center;
+  gap: 88px;
+}
```

Comment:
- [ ] Please pay attention to the design of the About section, particularly the curved background design, and color.🤔🤔
- [ ]  It is essential to adhere closely to the Figma design in order to make your app visually appealing and attract the attention of future recruiters.❤️❤️🎉
- [ ] your design
![your](https://github.com/Microverse-ajgomes/Personal-Portfolio/assets/118761303/2fde9c6b-665e-40ce-935f-208cf024efb3)

- [ ] Figma
![figma](https://github.com/Microverse-ajgomes/Personal-Portfolio/assets/118761303/a4b56b3b-965b-412c-ad5f-bdbdb5387bfa)